CHANGELOG.md

```diff
@@ -133,6 +133,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Prevent read beyond slice boundary in ByteArrayIndexInput ([#10481](https://github.com/opensearch-project/OpenSearch/issues/10481))
 - Fix the "highlight.max_analyzer_offset" request parameter with "plain" highlighter ([#10919](https://github.com/opensearch-project/OpenSearch/pull/10919))
 - Warn about deprecated and ignored index.mapper.dynamic index setting ([#11193](https://github.com/opensearch-project/OpenSearch/pull/11193))
+- Fix for `FloatField` back to `FloatPoint` in terms query ([#12499](https://github.com/opensearch-project/OpenSearch/pull/12499))
```

Comment:
Can you clarify the broken experience? 

`terms` query was broken on `float` fields that had `"doc_values":false`, right?

Reply:
updated this.